docs/technical-design/database-diagram.md

```diff
@@ -38,5 +40,44 @@ User {
     string role
 }
 
-User }|--|{ State: "many to many"
+Document {
+    string id
+    string name
+    string s3URL
+    string uploadedBy
```

Comment:
Should `uploadedBy` type be `User` or are we storing just the users email or id as a string?
```suggestion
    string uploadedBy
```